src/error_messages/error_messages.h

```diff
@@ -187,11 +187,11 @@
 #define REM_ERROR       "(1956): Error removing '%s' file."
 #define NEW_GLOB_FILE   "(1957): New file that matches the '%s' pattern: '%s'."
 #define DUP_FILE        "(1958): Log file '%s' is duplicated."
-#define FORGET_FILE     "(1959): File '%s' does not exist."
+#define FORGET_FILE     "(1959): File '%s' no longer exists. It will be forgotten."
```

Comment:
This message could be confusing in some cases when it remains more attempts to open the file again, here an example:
```
2018/12/12 09:40:03 ossec-logcollector[121168] logcollector.c:370 at LogCollectorStart(): INFO: (1959): File '/root/test/test.log' no longer exists. It will be forgotten.
2018/12/12 09:40:03 ossec-logcollector[121168] logcollector.c:385 at LogCollectorStart(): DEBUG: (1962): Unable to open file '/root/test/test.log'. Remaining attempts: 2
2018/12/12 09:40:08 ossec-logcollector[121168] logcollector.c:385 at LogCollectorStart(): DEBUG: (1962): Unable to open file '/root/test/test.log'. Remaining attempts: 1
2018/12/12 09:40:13 ossec-logcollector[121168] logcollector.c:385 at LogCollectorStart(): DEBUG: (1962): Unable to open file '/root/test/test.log'. Remaining attempts: 0
2018/12/12 09:40:13 ossec-logcollector[121168] logcollector.c:509 at LogCollectorStart(): INFO: (1904): File not available, ignoring it: '/root/test/test.log'.
```

Reply:
Fixed. I removed the text "It will be forgotten".